js/ui/gallery.js

```diff
@@ -638,25 +641,16 @@ const Gallery = CollectionWidget.inherit({
             this._releaseInvisibleItems();
             return;
         }
-
-        this._itemElements().each((function(index, item) {
-            if(this.option('selectedIndex') === index) {
-                $(item).removeClass(GALLERY_INVISIBLE_ITEM_CLASS);
-            } else {
-                $(item).addClass(GALLERY_INVISIBLE_ITEM_CLASS);
+        const selectedIndex = this.option('selectedIndex');
+        this._itemElements().each((index, item) => {
+            if(selectedIndex !== index) {
+                $(item).find(ITEM_CONTENT_SELECTOR).addClass(GALLERY_INVISIBLE_ITEM_CLASS);
```

Comment:
does the previous selected index get this class now?

Reply:
No, before call the '_renderItemVisibility' method always called '_releaseInvisibleItems' method that remove hidden class